rewrite-core/src/test/java/org/openrewrite/GitRemoteTest.java

```diff
@@ -79,6 +79,8 @@ void parseUnknownRemote(String cloneUrl, String expectedOrigin, String expectedP
       git@scm.company.com:stash/org/repo.git, scm.company.com/stash, Bitbucket, org/repo, org, repo
       ssh://scm.company.com/stash/org/repo, scm.company.com/stash, Bitbucket, org/repo, org, repo
       ssh://scm.company.com:22/stash/org/repo, scm.company.com/stash, Bitbucket, org/repo, org, repo
+      ssh://scm.company.com:22/stash/org/repo, scm.company.com:22/stash, Bitbucket, org/repo, org, repo
+      ssh://scm.company.com:7999/stash/org/repo, scm.company.com/stash, Bitbucket, org/repo, org, repo
       ssh://scm.company.com:7999/stash/org/repo, scm.company.com:7999/stash, Bitbucket, org/repo, org, repo
 
```

Comment:
aren't these the same inputs? expected different results?

Reply:
no, the origin is both input and expected result